requirements.txt

```diff
@@ -8,3 +8,4 @@ sympy>=1.2
 mock
 numpydoc
 nbsphinx
+harold==1.0.1
```

Comment:
This is different from the previous version requirement (>=1.0.1)

Reply:
`1.0.1` is the current version, so I figured to make it `>=` rather than `==` and only change it to `==` should we need to in the future.